### PR TITLE
HTML fixes

### PR DESCRIPTION
**Pull request by CyberShadow**

### CODE REVIEW

File: changelog/changelog.ddoc [hunk 1]
```diff
@@ -32,14 +32,14 @@ BUGSTITLE = <div class="bugsfixed">$(H4 $1) $(OL $2 )</div>
...
-DRUNTIMEBUGSFIXED2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2&query_format=advanced&chfield=resolution&chfieldfrom=$1&chfieldvalue=FIXED&bug_severity=regression&bug_severity=blocker&bug_severity=critical&bug_severity=major&bug_severity=normal&bug_severity=minor&bug_severity=trivial&bug_status=RESOLVED&version=D2&version=D1%20%26%20D2&component=druntime&resolution=FIXED&product=D, Druntime Bugs Fixed)
-PHOBOSBUGSFIXED2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2&query_format=advanced&chfield=resolution&chfieldfrom=$1&chfieldvalue=FIXED&bug_severity=regression&bug_severity=blocker&bug_severity=critical&bug_severity=major&bug_severity=normal&bug_severity=minor&bug_severity=trivial&bug_status=RESOLVED&version=D2&version=D1%20%26%20D2&component=Phobos&resolution=FIXED&product=D, Phobos Bugs Fixed)
+WHATSNEW2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2$(AMP)query_format=advanced$(AMP)chfield=resolution$(AMP)chfieldfrom=$1$(AMP)chfieldvalue=FIXED$(AMP)bug_severity=enhancement$(AMP)bug_status=RESOLVED$(AMP)version=D2$(AMP)version=D1%20%26%20D2$(AMP)resolution=FIXED$(AMP)product=D, New/Changed Features)
```
Comment (WalterBright) [inline changelog/changelog.ddoc]: Is it correct to do tis inside a URL?

Comment (CyberShadow) [inline changelog/changelog.ddoc]: Yes, all ampersands in HTML must be escaped unless they start an entity. `<script>` tags get a pass as a special exception, I think.

**Comment by JackStouffer** [comment]
LGTM

**Comment by AndrejMitrovic** [comment]
LGTM. Merge button is missing though.

@andralex can I get merge rights for dlang.org, bitte? :)

**Comment by andralex** [comment]
Thanks, good cleanup. Did you use a tool for that? If so, shall we use it automatically?

**Comment by CyberShadow** [comment]
> Did you use a tool for that? If so, shall we use it automatically?

That's the plan. This PR has been blocking automatic HTML validation in the Ddoc autotester :)

**Pull request merged**